charts/kspm-collector/CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ Manual edits are supported only below '## Change Log' and should be used
 exclusively to fix incorrect entries and not to add new ones.
 
 ## Change Log
+# v0.1.51
```

Comment:
CHANGLOG.md automatically updates, no need to edit it manually

Reply:
got it - removed my changes